app/client/ui/PagePanels.ts

```diff
@@ -532,6 +532,7 @@ const cssLeftPaneHeader = styled(cssHeader, `
 `);
 const cssRightPaneHeader = styled(cssHeader, `
   background-color: ${theme.rightPanelBg};
+  border-bottom: 0;
```

Comment:
The header borders are now handled on each tab styling, since the current tab is visually merged with the tab content.

This assumes we always have tabs in the right panel. Is that true?

Reply:
Document History doesn't have tabs, but it looks ok to me with your changes.